app/webpacker/controllers/toggle_button_disabled_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["button"];
+
+  connect() {
+    // Hacky way to go arount mrjus automatically enabling/disabling form element
+    setTimeout(() => {
+      if (this.hasButtonTarget) {
+        this.buttonTarget.disabled = true;
```

Comment:
I've noticed that reloading the page causes this controller to not work correctly in some cases, and re-enable the button. I'll have a look at it...

Reply:
Okay, there was a weird race condition here, and it turns out Rails was adding the `data-disabled-with` property by default (since Rails 7.0), which was partly the issue.

I'll send you a little PR in a sec...